openapi.yaml

```diff
@@ -208,10 +497,378 @@ definitions:
+    example: 1000
+  PlayerBet:
+    type: integer
+    minimum: 0
+    description: The bet from a specific player.
+  MemberEmail:
+    type: string
+    example: abc@efg.com
+    format: email
+  MemberExpAboveLevel:
+    type: integer
+    minimum: 0
+    description: Player's experience point above current level.
+    example: 1234
+  MemberId:
+    type: string
+    example: '101'
+  MemberIds:
+    type: array
+    items:
+      $ref: '#/definitions/MemberId'
+  PlayerId:
+    type: string
+    example: 'g1m1'
+  PlayerIds:
+    type: array
+    items:
+      $ref: '#/definitions/PlayerId'
+  PlayerInGamePatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          id:
+            $ref: '#/definitions/PlayerId'
+          type:
+            $ref: '#/definitions/PlayerType'
+          attributes:
+            type: object
+            properties:
+              playerBet:
+                $ref: '#/definitions/PlayerBet'
+              playerStatus:
+                $ref: '#/definitions/PlayerStatus'
+              playerCards:
+                $ref: '#/definitions/Cards'
+            additionalProperties: false
+        required:
+          - id
+          - type
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  PlayerInGameResource:
+    properties:
+      id:
+        $ref: '#/definitions/PlayerId'
+      type:
+        $ref: '#/definitions/PlayerType'
+      attributes:
+        type: object
+        properties:
+          memberId:
+            $ref: '#/definitions/MemberId'
+          memberName:
+            $ref: '#/definitions/MemberName'
+          memberLevel:
+            $ref: '#/definitions/MemberLevel'
+          memberExpAboveLevel:
+            $ref: '#/definitions/MemberExpAboveLevel'
+          playerBet:
+            $ref: '#/definitions/PlayerBet'
+          playerStatus:
+            $ref: '#/definitions/PlayerStatus'
+          playerCards:
+            $ref: '#/definitions/Cards'
+  PlayerInGameResult:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        $ref: '#/definitions/PlayerInGameResource'
+  PlayerInGameResults:
+    properties:
+      links:
+        $ref: '#/definitions/SelfLink'
+      data:
+        type: array
+        items:
+          $ref: '#/definitions/PlayerInGameResource'
+  PlayerPostBody:
+    properties:
+      data:
+        type: object
+        properties:
+          type:
+            $ref: '#/definitions/PlayerType'
+          attributes:
+            type: object
+            properties:
+              memberId:
+                $ref: '#/definitions/MemberId'
+            required:
+              - playerId
+            additionalProperties: false
+        required: 
+          - type
+          - attributes
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  PlayerResult:
```

Comment:
It looks like you have `PlayerResult` and `PlayerInGameResult` that are duplicates of each other. You should be able to just use one of these. Since there is only one `player` resource, it might be better to have these called `PlayerResult` and `PlayerResource` instead of `PlayerInGameResult` and `PlayerInGameResource`